bika/health/utils.py

```diff
@@ -163,3 +167,53 @@ def handle_after_submit(context, request, state):
     else:
         status_id = "success"
     return status_id
+
+
+def get_age_ymd(birth_date, to_date=None):
+    """Returns the age at to_date if not None. Otherwise, current age
+    """
+    delta = get_relative_delta(birth_date, to_date)
```

Comment:
If a patient does not have birthdate due to an old inconsistency, the system rises an "internal error" to the user without pointing out the source of the issue.

I think that the upgrade step should consider and fix patients without birth date.

Reply:
The Birth Date is a required field in senaite.health:
https://github.com/senaite/senaite.health/blob/master/bika/health/content/patient.py#L133
I think this is the correct behavior

Reply:
Right